Formula/docker-clean.rb

```diff
@@ -0,0 +1,19 @@
+# Docker-Clean Script
+# Maintained by Killian Brackey && Sean Kilgarriff at ZZROT Design
+# Contact us at docker@zzrot.com for any issues or fork and pull from
+# the Github repo below.  Happy cleaning!
```

Comment:
Please remove these comments; you can't actually maintain this as you don't have commit access 😉